terraform/TEMPLATES/_vc_details.html

```diff
@@ -1,5 +1,5 @@
 <turbo-frame id="vc">
-   <div class="container">
+   <!--<div class="container">
```

Comment:
Would you rather remove unused code instead of commenting it out. 
Best to just remove unused code.

Reply:
The reason I left is if we are gonna re-add this capability in the future assuming we will support not having to do linked clones. But I do see your point as well, I will remove it anyway is a simple piece of code.